### ISSUE TITLE
ToggleButton: aria-label and aria-labelledby not observed

### ISSUE THREAD

**Issue by jacobtylerwalls** [primefaces/primevue#6377]
### Describe the bug

ToggleButton is documented to take `aria-label` and `aria-labelledby` props, but nothing is done with them.

### Reproducer

https://stackblitz.com/edit/missing-aria-label?file=src%2FApp.vue

### PrimeVue version

4.0.1

### Vue version

4.x

### Language

TypeScript

### Build / Runtime

Vue CLI App

**Comment by fafeitsch** [comment]
Hey, thanks for the fast fix in 4.0.7. The attribute `arial-abelledby` does now work.

However, the aria-label still does not work like the documentation (https://primevue.org/togglebutton/#accessibility) says it should, it's still ignored. Your commit seems to have removed the aria-label completely from the ToggleButton. 

What was the rationale of removing the support for aria-label for the ToggleButton?
